cmd/hyperkube/hyperkube.go

```diff
@@ -123,8 +123,8 @@ func (hk *HyperKube) Run(args []string) error {
 	command := args[0]
 	baseCommand := path.Base(command)
 	serverName := baseCommand
+	args = args[1:]
 	if serverName == hk.Name {
-		args = args[1:]
```

Comment:
Ah!  Looking at the documentation for [`pflags.FlagSet.Parse`](https://godoc.org/github.com/spf13/pflag#FlagSet.Parse) it looks like the args should *not* contain the command name.  That would imply that we were doing it wrong before when the command name was being included.  `interspersed` defaulting to `true` must have covered our sins.